### ISSUE TITLE
Android 8 Support

### ISSUE THREAD

**Issue by nidhinprathap** [jayakrishnan-pm/LocationTrackerApplication#2]
As per the new Android Policies by google , you wouldnt get location updates after you go in background . Ulll get the updates once in 45 mins or something. To overcome this you need to start the Service as foreground activity 

https://developer.android.com/about/versions/oreo/background-location-limits

**Comment by Alfaizkhan** [comment]
Yes. We have to add persistent **notification** if we want to run background services.
